tools.js

```diff
@@ -535,11 +535,9 @@ module.exports = (function () {
 					banlistTable[subformat.banlist[i]] = subformat.name || true;
 					banlistTable[toId(subformat.banlist[i])] = subformat.name || true;
 
-					var plusPos = subformat.banlist[i].indexOf('+');
 					var complexList;
-					if (plusPos && plusPos > 0) {
```

Comment:
I have no idea what this line is supposed to be doing. To me it looks like whoever wrote it just didn't understand how `indexOf()` works, but because it's so weird I'm just making sure I didn't change some guard against undefined variables or something.